lang/cs/Org.Apache.REEF.Driver/Evaluator/EvaluatorDescriptorImpl.cs

```diff
@@ -39,15 +39,16 @@ public sealed class EvaluatorDescriptorImpl : IEvaluatorDescriptor
         private readonly int _megaBytes;
         private readonly INodeDescriptor _nodeDescriptor;
         private readonly string _rack;
+        private readonly string _runtimeName;
 
-        internal EvaluatorDescriptorImpl(INodeDescriptor nodeDescriptor, EvaluatorType type, int megaBytes, int core,
-            string rack = DefaultRackName)
+        internal EvaluatorDescriptorImpl(INodeDescriptor nodeDescriptor, EvaluatorType type, int megaBytes, int core, string rack = DefaultRackName, string runtimeName = "")
         {
             _nodeDescriptor = nodeDescriptor;
             _evaluatorType = type;
             _megaBytes = megaBytes;
             _core = core;
             _rack = rack;
+            _runtimeName = runtimeName;
         }
 
         /// <summary>
```

Comment:
side note: not entirely relevant to this PR, but I was wondering if a ``static`` "factory" method is the better way to go here... I don't particularly like initializing the variables in two places, and we can't use ``this()`` since ``this()`` will call the ``internal`` constructor first.